config/rubocop.yml

```diff
@@ -19,15 +19,14 @@ Style/StringLiterals:
 Metrics/LineLength:
   Max: 100
```

Comment:
Aside from the specs in the `Exclude` block, do you know what the max is? I might be inclined to tighten this a bit and see what the absolute maximum is; and then everytime you touch an area of the rubocop config attempt to raise the bar a tiny bit.